Arduino/tda7719-library-mod/TDA7719.cpp

```diff
@@ -7,27 +7,18 @@ TDA7719::TDA7719(){
 }
 
 void TDA7719::sendConfig(uint8_t* data) {
-    writeWire(TDA7719_INPUT, data[0]);
-    writeWire(TDA7719_INPUT_2, data[1]);
-    writeWire(TDA7719_MIX_SOUR, data[2]);
-    writeWire(TDA7719_MIX_CONT, data[3]);
-    writeWire(TDA7719_MUTE, data[4]);
-    writeWire(TDA7719_SOFT_1, data[5]);
-    writeWire(TDA7719_SOFT_2, data[6]);
-    writeWire(TDA7719_LOUD, data[7]);
-    writeWire(TDA7719_VOL, data[8]);
-    writeWire(TDA7719_TRBLE, data[9]);
-    writeWire(TDA7719_MIDDLE, data[10]);
-    writeWire(TDA7719_BASS, data[11]);
-    writeWire(TDA7719_SUB_M_B, data[12]);
-    writeWire(TDA7719_ATT_LF, data[13]);
-    writeWire(TDA7719_ATT_RF, data[14]);
-    writeWire(TDA7719_ATT_LR, data[15]);
-    writeWire(TDA7719_ATT_RR, data[16]);
-    writeWire(TDA7719_ATT_SUB_L, data[17]);
-    writeWire(TDA7719_ATT_SUB_R, data[18]);
-    writeWire(TDA7719_TEST_1, data[19]);
-    writeWire(TDA7719_TEST_2, data[20]);
+
+        for (size_t i = 0; i < size; i++) {
+                writeWire(i, data[i]); 
+        }
```

Comment:
previous version is easier to understand, since not a very very long function that length seems ok